### ISSUE TITLE
Question about configure tpot

### ISSUE THREAD

**Issue by dpicollege** [dtag-dev-sec/tpotce#94]
Hello,
I Have two question
1- which file i should configure to keep all data and where is the file exactly?
2- I deploy 3 instance of tpot in EU  but when i correlated all the log it's just show me sensor name is "tpot" so how can i change the sensor name to " tpot-french" for example in kibana? which file i should change?
Thanks

**Comment by t3chn0m4g3** [comment]
Regarding 1 you have to modify the systemd configs and turn persistence on, i.e.:
```
/etc/systemd/system/dionaea.service
[...]
ExecStartPre=/bin/bash -c '/usr/bin/clean.sh dionaea on'
[...]
```
Regarding 2, currently not implemented but is already a roadmap item for 17.06 release.

**Comment by mattmac1** [comment]
Feed the data to a separate elasticsearch or something because if you've
got three separate instances you would want to have the output in one place
anyway no?
